php/auth.php

```diff
@@ -0,0 +1,49 @@
+<?php
+$ERRORLOG = getenv('PHP_ERROR_LOG');
```

Comment:
Good call, I forgot to add a default at this layer so core pi-hole doesn't need to worry about php ENVs.  My docker does that for me :(  I'll add some checking logic to set a default if not found as an ENV/SERVER var